packages/headless/src/features/analytics/analytics-utils.ts

```diff
@@ -71,6 +82,100 @@ export interface AsyncThunkAnalyticsOptions<
   extra: ThunkExtraArguments;
 }
 
+export interface temp__MakeAnalyticsActionReturnType<T extends AnalyticsType> {
```

Comment:
If both are actions it can probably be confusing, I would rename to what they do
- updateAnalyticsAction or something similar instead of description
- logAnalyticsAction or something similar instead of just action